### PR TITLE
adding tesnet 34 details to sporks.json

### PR DESCRIPTION
**Pull request by vishalchangrani**
Closes: #???

## Description

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: sporks.json [hunk 1]
```diff
@@ -862,9 +862,55 @@
...
+        "rootStateCommitment": "f2e77e16628543e285be8bac677db06e17b37f49e53a107af1e6bf99fbc17b30",
+        "gitCommitHash": "32bda868ae651cd792f7f78dee31911141c5da8e",
+        "stateArtefacts": {
+          "gcp": {
+            "rootCheckpointFile": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34-execution/public-root-information/root.checkpoint",
+            "rootProtocolStateSnapshot": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34/public-root-information/root-protocol-state-snapshot.json",
+            "rootProtocolStateSnapshotSignature": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34/public-root-information/root-protocol-state-snapshot.json.asc",
+            "nodeInfo": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34/public-root-information/node-infos.pub.json",
+            "executionStateBucket": "flow_public_devnet34_execution_state",
+            "protocolDBArchive": "https://storage.googleapis.com/flow-execution-archive/devnet34/protocol-db.tar.gz",
```
Comment (vishalchangrani) [inline sporks.json:881]: where does this come from? The values for the following are not correct and need to be updated but I am not sure where they come from?

> protocolDBArchive
> protocolDBArchiveChecksum
> executionStateArchive
> executio

nStateArchive

File: sporks.json [hunk 2]
```diff
@@ -862,9 +862,55 @@
...
+          "s3": {
+            "rootCheckpointFile": "",
+            "rootProtocolStateSnapshot": "",
+            "nodeInfo": "",
+            "executionStateBucket": ""
+          }
+        },
+        "tags": {
+          "flow-go-tag": "v0.25.6",
+          "flow-dps-tag": "v0.24",
```
Comment (vishalchangrani) [inline sporks.json:895]: @m4ksio where does this value come from?

File: sporks.json (hunk 1 above)
Comment (peterargue) [reply sporks.json:881]: you can leave these empty. They are for buckets where we've uploaded the protocol and execution state for historic sporks. We may populate them in the future

File: sporks.json [hunk 3]
```diff
@@ -862,9 +862,55 @@
...
+            "address": "access-004.devnet34.nodes.onflow.org:3570",
+            "key": "0d1523612be854638b985fc658740fa55f009f3cd49b739961ab082dc91b178ed781ef5f66878613b4d34672039150abfd9c8cfdfe48c565bca053fa4db30bec"
+          }
+        ]
+      },
       "testnet33": {
         "id": 33,
-        "live": true,
+        "live": false,
         "name": "Testnet32",
```
Comment (peterargue) [inline sporks.json:914]: while you're in here. can you update this
```suggestion
        "name": "Testnet33",
```

File: sporks.json (hunk 1 above)
Comment (vishalchangrani) [reply sporks.json:881]: thanks!

**Review by franklywatson — APPROVED** [review]
Not sure about the dps-tag but pretty sure it's not being used.

File: sporks.json (hunk 2 above)
Comment (m4ksio) [reply sporks.json:890]: But since we don't have it I think we can leave it blank for now

**Review by koko1123 — APPROVED** [review]

**Pull request merged**